app/assets/stylesheets/_buttons.scss

```diff
@@ -1,265 +1,159 @@
+  -webkit-transition-timing-function: cubic-bezier(0.7,0,0.3,1);
+  transition-timing-function: cubic-bezier(0.7,0,0.3,1);
+  -webkit-transform: scale3d(0,0,1);
+  transform: scale3d(0,0,1);
 }
 
 @mixin translate-vertical($pixels) {
-    -webkit-transform: translate3d(0, $pixels ,0);
-    transform: translate3d(0, $pixels ,0);
-}
-
-
-.tabs-style-circle {
-    .tabs-row {
-        padding-top: 50px;
-        padding-bottom: 50px;
-    }
-	overflow: visible;
-    li::before {
-        position: absolute;
-        top: 50%;
-        left: 50%;
-        margin: -60px 0 0 -60px;
-        width: 120px;
-        height: 120px;
-        border: 1px solid $medium-green;
-        border-radius: 50%;
-        content: '';
-        opacity: 0;
-        @include circle-transition();
-    }
-    li {
-        a {
-            color: $charcoal;
-        }
-    }
-    li.tab-current {
-        a {
-            color: $medium-green;
-            span {
-                //@include translate-vertical(4px);
-            }
-        }
-        &::before {
-            opacity: 1;
-            -webkit-transform: scale3d(1,1,1);
-            transform: scale3d(1,1,1);
-        }
-        .icon::before {
-            @include translate-vertical(-4px);
-        }
-    }
-    a {
-        overflow: visible;
-        color: $charcoal;
-        font-weight: 400;
-        font-size: 0.9em;
-        line-height: 1.1;
-        @include cubic-bezier-color();
-        span {
-            display: inline-block;
-            @include cubic-bezier-transform();
-        }
-    }
-    .icon::before {
-        @include cubic-bezier-transform();
-        display: block;
-        margin: 0;
-        pointer-events: none;
-    }
-    a:hover, a:focus {
-        color: $medium-green;
-    }
-}
-
-@media #{$small-only}  {
-	.tabs-style-circle li::before {
-		margin: -40px 0 0 -40px;
-		width: 80px;
-		height: 80px;
-	}
+  -webkit-transform: translate3d(0, $pixels ,0);
+  transform: translate3d(0, $pixels ,0);
 }
 
 @mixin icon($path) {
-    width: 20px;
-    height: 20px;
-    background-size: 100%;
-    background-image: url($path);
-    position: relative;
-    display: block;
+  width: 20px;
+  height: 20px;
+  background-size: 100%;
+  background-image: url($path);
+  position: relative;
+  display: block;
 }
 
 @mixin button-transition {
-    -webkit-transition: -webkit-transform 0.5s ease;
-    -moz-transition: -moz-transform 0.5s ease;
-    -o-transition: o-transform 0.5s ease;
-    transition: all 0.5s ease;
+  -webkit-transition: -webkit-transform 0.5s ease;
+  -moz-transition: -moz-transform 0.5s ease;
+  -o-transition: o-transform 0.5s ease;
+  transition: all 0.5s ease;
 }
 
 .material-success-button {
-    @extend .button;
-    @include button-transition();
-    @include drop-shadow(1);
+  @extend .button;
+  @include button-transition();
+  @include drop-shadow(1);
 
-    text-transform: uppercase;
-    border-radius: $global-radius;
+  text-transform: uppercase;
```

Comment:
Properties should be ordered background-color, border, border-radius, text-transform